.github/workflows/PR-build.yml

```diff
@@ -57,52 +57,59 @@ jobs:
     # Set up building environment, patch the dev repo code on dispatch events.  
     - name: Set up Go 1.x
       uses: actions/setup-go@v2
+      if: ${{ needs.changes.outputs.changed == 'true' }}
```

Comment:
This is just better formatting? The functionality is the same?

Reply:
Yes. It's mostly for formatting purpose.